docker-compose.yml

```diff
@@ -24,14 +26,16 @@ services:
     image: selenium/standalone-edge:4.10.0-20230607
     container_name: selenium-container
     network_mode: host
+    ports:
+      - "4444:4444"
     healthcheck:
-      test: ["CMD", "curl", "-f", "http://localhost:4444"]
-      interval: 30s
-      timeout: 10s
+      test: ["CMD", "curl", "-sS", "-f", "http://localhost:4444"]
+      interval: 60s
```

Comment:
Curious as to why the longer interval as opposed to increasing retries?

Reply:
During testing, I had initially set the interval to 60s, but after reviewing the configuration, I’ve pushed the changes with the standard interval and retries. There’s no need for the longer interval, so I’ve reverted it back.